engine/settings/base.py

```diff
@@ -106,7 +106,7 @@ class DatabaseTypes:
 DATABASE_CONFIGS = {
     DatabaseTypes.SQLITE3: {
         "ENGINE": DATABASE_ENGINE,
-        "NAME": DATABASE_NAME or "/var/lib/oncall/oncall.db",
```

Comment:
I think `or "/var/lib/oncall/oncall.db"` should stay so there's a sensible default value and there's no need to specify `DATABASE_NAME` when using sqlite.
Probably this also would break existing hobby instances that don't declare `DATABASE_NAME` explicitly.